ui/site/css/_ask.scss

```diff
@@ -0,0 +1,380 @@
+$gap: 6px;
```

Comment:
here's a lot more scss to maintain. Better hope we never go through a redesign or responsive relayout. Or tech change.